searchcore/src/vespa/searchcore/proton/metrics/executor_metrics.cpp

```diff
@@ -7,16 +7,19 @@ namespace proton {
 void
 ExecutorMetrics::update(const vespalib::ThreadStackExecutorBase::Stats &stats)
 {
-    maxPending.set(stats.maxPendingTasks);
+    maxPending.set(stats.queueSize.max());
     accepted.inc(stats.acceptedTasks);
     rejected.inc(stats.rejectedTasks);
+    const vespalib::ThreadStackExecutorBase::Stats::QueueSizeT & qSize = stats.queueSize;
+    queueSize.addValueBatch(qSize.average(), qSize.count(), qSize.min(), qSize.max());
 }
 
 ExecutorMetrics::ExecutorMetrics(const std::string &name, metrics::MetricSet *parent)
     : metrics::MetricSet(name, {}, "Instance specific thread executor metrics", parent),
       maxPending("maxpending", {}, "Maximum number of pending (active + queued) tasks", this),
       accepted("accepted", {}, "Number of accepted tasks", this),
-      rejected("rejected", {}, "Number of rejected tasks", this)
+      rejected("rejected", {}, "Number of rejected tasks", this),
+      queueSize("queuesize", {}, "size of task queue", this)
```

Comment:
"size" -> "Size"

Reply:
Fixed in https://github.com/vespa-engine/vespa/pull/12711/commits/a4f448633278360f7cb8ef2135f68291cb021a9f